src/App/components/Accordion.scss

```diff
@@ -9,10 +9,25 @@
       margin-bottom: 15px;
     }
   }
-  &__card-title {
+  &__card-title-button  {
+    width: 100%;
+    background: rgba(0,0,0,0.05) !important;
```

Comment:
Is the `!important` strictly necessary (here and below). It makes maintaining CSS difficult.

Reply:
Yes...turns out I found a bug in blueprint3's dark theme that is already documented and added to their list to fix so their default CSS will prevent this unless I force it.

Reply:
Once blueprint3 fixes the bug we can delete this custom css and the code that applies it and then the intent="success" on the .js itself will handle the rest.